pkgs/servers/gemini/agate/default.nix

```diff
@@ -23,9 +23,11 @@ rustPlatform.buildRustPackage rec {
     runHook postInstallCheck
   '';
 
+  passthru.tests = { inherit (nixosTests) agate; };
+
   meta = with lib; {
-    homepage = "https://proxy.vulpes.one/gemini/qwertqwefsday.eu/agate.gmi";
-    changelog = "https://proxy.vulpes.one/gemini/qwertqwefsday.eu/agate.gmi";
+    homepage = "gemini://qwertqwefsday.eu/agate.gmi";
```

Comment:
This link should be visitable from search.nixos.org.

Reply:
We went over this previously where I put this proxy for you and then it died.

They put their homepage has this gemini address so I insist it stays
It is not weird to expect someone looking at gemini servers to have a gemini client.
There are also ftp links and as mentioned previously a gopher link

Reply:
why not just the github?

Reply:
Also, not everyone who nixpkgs uses gemini. The meta values should be a "lowest common denominator" for accessibility.